h2o-assembly/build.gradle

```diff
@@ -0,0 +1,137 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+import java.util.zip.ZipEntry
+import java.util.zip.ZipInputStream
+import java.util.zip.ZipOutputStream
+
+apply plugin: 'com.github.johnrengelman.shadow'
+apply plugin: 'base'
+
+description = """
+H2O and H2O Driver Assembly. This assembly extends provided jar by sparkling-water specific classes needed in external
+backend mode.
+
+User can specify location to either h2o or h2odriver jar via H2O_ORIGINAL_JAR env variable.
+No works is done if H2O_ORIGINAL_JAR is not set.
+
+User can also use gradle command line argument -PdownloadAndExtend which will download correct h2o version for current
+sparkling water from our repository if it hasn't been already downloaded.
+If this is run without any argument, then basic H2O.jar is downloaded,
+but this command takes also argument specifying hadoop version, such as cdh5.4. In case the argument is specified, instead
+of downloading H2O.JAR this command will download h2o driver jar for specified hadoop version.
+
+Gradle property has higher priority if both H2O_ORIGINAL_JAR and -PdownloadAndExtend are set
+
+"""
+
+configurations{
+  artifactsToMerge
+}
+
+ext.supportedHadoopVersions = ["cdh5.2","cdh5.3", "cdh5.4", "cdh5.5", "cdh5.6", "cdh5.7", "cdh5.8", "hdp2.1",
```

Comment:
You do not need to list them here, since it is hard to synchronize the list with actual list in H2O. Just keep it simple - if user specify wrong version, the script reports error